libsolidity/interface/Natspec.cpp

```diff
@@ -36,6 +36,10 @@ Json::Value Natspec::userDocumentation(ContractDefinition const& _contractDef)
 	Json::Value doc;
 	Json::Value methods(Json::objectValue);
 
+	auto dev = extractDoc(_contractDef.annotation().docTags, "notice");
```

Comment:
Why is the variable called `dev`? Shouldn't it be `notice`? Also below in the loop we use an explicit type (`string`).

Reply:
Fixed!